packages/wdio-browserstack-service/src/@types/bstack-service-types.d.ts

```diff
@@ -1,11 +1,13 @@
 declare namespace WebdriverIO {
     interface Browser {
         getAccessibilityResultsSummary: () => Promise<{ [key: string]: any; }>,
-        getAccessibilityResults: () => Promise<Array<{ [key: string]: any; }>>
+        getAccessibilityResults: () => Promise<Array<{ [key: string]: any; }>>,
+        percyCaptureMap: any
     }
 
     interface MultiRemoteBrowser {
         getAccessibilityResultsSummary: () => Promise<{ [key: string]: any; }>,
-        getAccessibilityResults: () => Promise<Array<{ [key: string]: any; }>>
+        getAccessibilityResults: () => Promise<Array<{ [key: string]: any; }>>,
+        percyCaptureMap: any
```

Comment:
If this is suppose to be used by users, we should type it correctly or remove it

Reply:
Typed it correctly to PercyCaptureMap